functional-test/src/test/groovy/org/rundeck/tests/functional/selenium/execution/CommandOnMultipleNodesSpec.groovy

```diff
@@ -0,0 +1,87 @@
+package org.rundeck.tests.functional.selenium.execution
+
+import org.rundeck.util.annotations.SeleniumCoreTest
+import org.rundeck.util.container.SeleniumBase
+import org.rundeck.util.gui.pages.execution.CommandPage
+import org.rundeck.util.gui.pages.execution.ExecutionShowPage
+import org.rundeck.util.gui.pages.login.LoginPage
+
+@SeleniumCoreTest
+class CommandOnMultipleNodesSpec extends SeleniumBase{
+
+    public static final String TEST_PROJECT = "core-jsch-executor-test"
+    public static final String TEST_SSH_ARCHIVE_DIR = "/projects-import/core-jsch-executor-test"
+    public static final String NODE_KEY_PASSPHRASE = "testpassphrase123"
+    public static final String NODE_USER_PASSWORD  = "testpassword123"
+    public static final String USER_VAULT_PASSWORD = "vault123"
+
+    @Override
+    void startEnvironment() {
+        String keyPath = getClass().getClassLoader().getResource("docker/compose/oss").getPath()+"/keys"
```

Comment:
this seems to be very specific for OSS env, if thats the case should we exclude this one from running on pro?